.travis.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+set -e -x
+
+make -C control check
```

Comment:
why is this needed? I mean there are many PR's without this .travis.sh (make -C ... respectively), so I'd expect at least some comment why it is done this way or (better) refactoring it into the generic script.